app/views/reports/index.html.erb

```diff
@@ -0,0 +1,43 @@
+<section class="content-header">
+  <div class="container-fluid">
+    <div class="row mb-2">
+      <div class="col-sm-6">
+        <% content_for :title, "Reports - #{current_organization.name}" %>
+        <h1>
+          Reports
+          <small>for <%= current_organization.name %></small>
+        </h1>
+      </div>
+      <div class="col-sm-6">
+        <ol class="breadcrumb float-sm-right">
+          <li class="breadcrumb-item"><%= link_to(dashboard_path) do %>
+              <i class="fa fa-dashboard"></i> Home
+            <% end %>
+          </li>
+          <li class="breadcrumb-item"><a href="#">Reports</a></li>
+        </ol>
+      </div>
+    </div>
+  </div>
+</section>
+<section class="content">
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-md-12">
+        <h5 class="mb-3">Reports are available at the end of every year. <%= "#{@actual_year} will be ready in #{@month_remaing_to_report} months!" %></h5>
```

Comment:
Hmm I am thinking maybe it makes sense to put another row that says:
```
#{actual_year} (available in #{@month_remaing_to_report})
```
and make this gray? 

I feel like users will read this more than the label as shown. Could you make that change please?